src/commands/signzone.rs

```diff
@@ -474,20 +492,48 @@ impl SignZone {
         let (apex, ttl) = Self::find_apex(&records).unwrap();
 
         // Hash the zone with NSEC or NSEC3.
+        let pb = if matches!(&log, Some(log) if log.is_terminal() && self.progress) {
+            let pb = ProgressBar::new(0).with_message("Hashing");
```

Comment:
This should probably be `ProgressBar::no_length()` and then with slightly different parsing. Also we should call `tick()` to make this actually show up, because it doesn't seem to do that by itself.

Reply:
I tried calling `tick()` but that also didn't help.